README.md

```diff
@@ -124,6 +124,10 @@ There is document symbol for Camel Contexts and routes, on XML and Java DSL. In
 ![Go to Symbol](./images/goToSymbol.gif "Go To Symbol")
 ![Breadcrumb](./images/breadCrumbXml.gif "Breadcrumb")
 
+## Connected mode completion for Kafka component
+
+When using the Camel Kafka component, the list of topics is dynamically retrieved from the Kafka Broker when available. By default, it tries to connect to `localhost:9092`. it can be connected to another instance using the system property `CAMEL_LANGUAGE_SERVER_KAFKA_CONNECTION_URL`.
```

Comment:
This seems a little confusing to me - "it can be connected to another instance using the system property" - may be slightly clearer if you say `By default, it tries to connect to `localhost:9092`. To connect to another instance, update the system property `CAMEL_LANGUAGE_SERVER_KAFKA_CONNECTION_URL`.`? Maybe provide an example of using that property somehow?